aslam_offline_calibration/kalibr/python/kalibr_calibrate_cameras

```diff
@@ -389,54 +390,54 @@ def main():
             print "Calibration complete."
             print
             if parsed.removeOutliers:
-                sm.logWarn("Removed {0} outlier corners.".format(len(removedOutlierCorners)) )             
+                sm.logWarn("Removed {0} outlier corners.".format(len(removedOutlierCorners)) )
             print
             print "Processed {0} images with {1} images used".format(numViews, calibrator.estimator.getNumBatches())
             kcc.printParameters(calibrator)
-            
+
             if parsed.verbose and len(calibrator.baselines)>1:
 		        f=pl.figure(100006)
 		        kcc.plotCameraRig(calibrator.baselines, fno=f.number, clearFigure=False)
 		        pl.show()
-            
+
             #write to file
             bagtag = parsed.bagfile.translate(None, "<>:/\|?*").replace('.bag', '', 1)
-            resultFile = "camchain-" + bagtag + ".yaml"
+            resultFile = parsed.output_folder + "/" + "camchain" + ".yaml"
```

Comment:
`resultFile = os.path.join(parsed.output_folder, "camchain" + ".yaml")`

and same for nearby lines